pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -251,6 +255,17 @@ func (s *Server) initConfigSources(args *PilotArgs) (err error) {
 					IstioRevision: args.Revision,
 				}.ToStruct(),
 				InitialDiscoveryRequests: adsc.ConfigInitialRequests(),
+				GrpcOpts: []grpc.DialOption{
+					grpc.WithKeepaliveParams(keepalive.ClientParameters{
```

Comment:
This is missing MaxConnectionAge. Can we make a function that converts `pkg/keepalive.Options` to gRPC option so it is used consistent?

Reply:
The MaxConnectionAge is used for grpc server. Here we configure the grpc client,  so we only care about the keepalive interval and timeout.
Add a function for pkg/keepalive.Options to convert it to grpc client option used for grpc client keepalive.